src/sdam/srv_polling/test.rs

```diff
@@ -182,7 +203,22 @@ async fn srv_max_hosts_random() {
         localhost_test_build_10gen(27020),
     ];
 
-    let actual = run_test_extra(Some(2), Ok(hosts)).await;
+    let actual = run_test_extra(Some(2), Ok(hosts), None).await;
     assert_eq!(2, actual.len());
     assert!(actual.contains(&localhost_test_build_10gen(27017)));
 }
+
+#[tokio::test]
+async fn srv_service_name() {
+    let hosts = vec![
+        localhost_test_build_10gen(27019),
+        localhost_test_build_10gen(27020),
+    ];
+    run_test_srv(
```

Comment:
`run_test_srv` just returns the addresses on the topology, so we'll need to add assertions on what gets returned here (see some of the other tests for examples)